setup.py

```diff
@@ -172,7 +172,7 @@ def run(self):
         'setuptools>=19.6',
         'sympy',
         'unyt',
-        'yt>=3.6',
+        'yt>=4.0.1',
```

Comment:
While you're at it, might as well future-proof it ?
```suggestion
        'yt>=4.0.1<5.0',
```